src/index.html

```diff
@@ -40,20 +40,7 @@
     href="https://fonts.googleapis.com/css2?family=Noto+Serif&family=Poppins:wght@300;400;500;700;900&display=swap"
     as="style" onload="this.onload=null;this.rel='stylesheet'" />
   <link rel="preload" href="/fonts/the-bold-font.css" as="style" onload="this.onload=null;this.rel='stylesheet'" />
-  <noscript>
-    <link
-      rel="preload"
-      href="https://fonts.googleapis.com/css2?family=Baloo+Da+2&family=Poppins:wght@300;400;500;700;900&display=swap"
-      as="style"
-      onload="this.onload=null;this.rel='stylesheet'"
-    />
-    <noscript>
```

Comment:
We still want a `<noscript>` for the fonts we use, otherwise fonts won't load at all with javascript is disabled.
Should be as simple as:
```html
<noscript>
  <link
    href="https://fonts.googleapis.com/css2?family=Noto+Serif&family=Poppins:wght@300;400;500;700;900&display=swap"
    rel="stylesheet"
  />
  <link
    href="/fonts/the-bold-font.css"
    rel="stylesheet"
  />
</noscript>
```